test/objects.jl

```diff
@@ -16,7 +16,13 @@ using HDF5.API
     h5open(fn, "r") do h5f
         @test API.h5o_exists_by_name(h5f, "data")
         @test API.h5o_exists_by_name(h5f, "lore")
-        @test_throws API.H5Error API.h5o_exists_by_name(h5f, "noonian")
+        @static if HDF5.API.h5_get_libversion() <= v"1.14.5"
```

Comment:
@derobins Are there no further planned patch releases on 0.14? Would it be safer to use
```suggestion
        @static if HDF5.API.h5_get_libversion() <= v"2"
```

Reply:
If there were another release of 1.14, it would include this bugfix, so >= 1.14.5 is probably safer.